src/game/SpellEffects.cpp

```diff
@@ -1151,7 +1151,7 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
                 }
                 case 32027:                                 // Expedition Flare
                 {
-                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
+                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
                         m_caster->CastSpell(m_caster, (urand(0, 1) ? 32029 : 32030), true, m_CastItem);
```

Comment:
It wont be always nullptr, consuming is done at the end of spell::cast if I remember correctly. Problem is that during next spell execution, lets say when its with a hidden cast time, the effect of the next spell might not necessarily happen immediately, hence the m_castItem will be nullptr inevitably. (due to the nature of how spellEvent scheduling works)